modules/programs/ssh/default.nix

```diff
@@ -81,11 +90,14 @@ in
       message = "knownHost ${name} must contain either a publicKey or publicKeyFile";
     });
 
-    environment.etc."ssh/ssh_known_hosts".text = (flip (concatMapStringsSep "\n") knownHosts
-      (h: assert h.hostNames != [];
-        concatStringsSep "," h.hostNames + " "
-        + (if h.publicKey != null then h.publicKey else readFile h.publicKeyFile)
-      )) + "\n";
+    environment.etc."ssh/ssh_known_hosts".text = ''
+      ${(flip (concatMapStringsSep "\n") knownHosts
+        (h: assert h.hostNames != [];
+          concatStringsSep "," h.hostNames + " "
+          + (if h.publicKey != null then h.publicKey else readFile h.publicKeyFile)
+        )) + "\n"}
+      ${cfg.extraConfig}
```

Comment:
I'm not sure how it's called in nixos, but this sounds a bit general given that it's only the known_hosts file.

Reply:
I actually noticed this was missing while trying to reuse an internal module from one of my nixos machines. Here is what nixos ssh module looks like https://github.com/NixOS/nixpkgs/blob/83fd9785f65b8fd4cbaa146270f90b2a0d74b5f3/nixos/modules/programs/ssh.nix#L217

Reply:
Yeah, but that's `ssh_config` not `known_hosts`.

Reply:
Whoops wrong config

Reply:
moved to `ssh_config`